src/shared/state/Applet/index.ts

```diff
@@ -28,4 +28,12 @@ export const applet = {
         },
       }) => data,
     ),
+  useAppletStatus: (): AppletSchema['applet']['status'] =>
```

Comment:
maybe rename to just useStatus, it will be used as `applet.useStatus`

Reply:
done